src/quadratic.jl

```diff
@@ -0,0 +1,72 @@
+"""
+Helper function for quadratic_interval that computes roots of a
+real quadratic using interval arithmetic to bound rounding errors.
+"""
+function quadratic_helper!{T}(a::Interval{T}, b::Interval{T}, c::Interval{T}, L::Array{Interval{T}})
+
+    Δ = b*b - 4*a*c
```

Comment:
Is there a reason to use `b*b` rather than `b^2` ? If there is it would be good to add a small comment explaining why, since the two expressions are not equivalent in general.

Reply:
`b^2` should be better here as it gives better benchmarks